mentors/go/w.json

```diff
@@ -14,5 +14,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I'm a former teacher, now an engineer at Pivotal. I'm passionate about learning new programming languages, and helping others learn."
-  }
+  },
+  {
+  "github_username": "walaaayyad",
+  "name": "Walaa Ayyad",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "I'm a front-end web developer, I'm passionate to learn new skills, and helping others to learn."
+}
```

Comment:
} should be indented two spaces.  Lines above should indented two more spaces until {.